pages/watches-support.json

```diff
@@ -422,13 +446,14 @@
       "features": [
         { "name":"Display", "status":"good" },
         { "name":"Touch", "status":"good" },
-        { "name":"Microphone", "status":"bad" },
         { "name":"Bluetooth", "status":"bad" },
-        { "name":"Compass", "status":"bad" },
         { "name":"Haptics", "status":"good" },
+        { "name":"Tilt-to-Wake", "status":"good" },
+        { "name":"Always-on-Display", "status":"good" },
+        { "name":"Microphone", "status":"bad" },
```

Comment:
The microphone is actually functional on `swift`: https://github.com/AsteroidOS/meta-swift-hybris/pull/9